### DISCUSSION TITLE
Extend the notification endpoint or add a new endpoint to display popups or toasts

### DISCUSSION THREAD

**Discussion by skuhnow** [directus/directus#17345] [Feature Requests]
Background: We use some custom hooks on items.create/update/delete, and we want to inform the user if anything fails. I found the `notifications/` endpoint, but the number next to the bell in the lower left is very inconspicuous.

Maybe an option `show_as_dialog` or `show_as_toast`, to trigger it. An alternative would be to use the existing toast notification, that is used to show a successful run of a manual flow (in the lower right corner of the app)

Sadly, I haven't found any workarounds with other types of extensions or flows

**Comment by skuhnow** [comment]
related #13727

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
